flow/flow.go

```diff
@@ -1024,6 +1024,10 @@ func (f *FlowLayer) GetFieldInt64(field string) (int64, error) {
 	switch field {
 	case "ID":
 		return f.ID, nil
+	case "A":
```

Comment:
could you add a functional test for that, maybe just re-using a test just add a GT/LT on port